src/test/java/jaiden/task/EventTest.java

```diff
@@ -0,0 +1,24 @@
+package jaiden.task;
+
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDate;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
```

Comment:
Perhaps reconsider the ordering of your import statements to be consistent and ensure they comply with the Java coding standards?
```suggestion
import static org.junit.jupiter.api.Assertions.assertEquals;
```